### DISCUSSION TITLE
How do I know which dependencies there are for plugins? And how to fix?

### DISCUSSION THREAD

**Discussion by Bug-Ninja** [DeaDBeeF-Player/deadbeef#2751] [Q&A]
I am a bit new to linux and so far I like DeaDBeeF  a lot. However, when I start DB there are 4 plugins that couldn't be properly loaded: 

```
plugin File Browser failed to connect to dependencies, deactivated.
plugin Converter GTK3 UI failed to connect to dependencies, deactivated.
plugin Playlist browser GTK3 failed to connect to dependencies, deactivated.
plugin Shellexec GTK3 UI failed to connect to dependencies, deactivated.
```
I would like to fix this, as Deadbeef crashes when I use the file browser for example. How do I know what the dependencies are? Are those in the repositories of my linux distro? (Mint 20)

Thank you for helping out a newbie!

**Comment by Oleksiy-Yakovenko** [comment]
File browser is an old 3rd party plugin, which contains code which causes deadbeef to crash.
There's nothing that can be done on deadbeef side, and the plugin needs fixing -- but the plugin has been abandoned by its developer, and nobody picked it up to fix. The only thing you can do is to remove this plugin.

The log messages: this was reported as a bug several times, but I can't remember whether this was fixed or not.
However, I assure you that these messages are harmless, and don't have anything to do with the crashes, and can be safely ignored.

And lastly, this question "How do I know what the dependencies are? Are those in the repositories of my linux distro? (Mint 20)"

The log messages are about plugin dependencies within deadbeef. It has nothing to do with package dependencies in your distro

**Comment by Bug-Ninja** [comment]
Thank you for your explanation! Too bad FB isn't developed anymore, but I understand you can't fix this.
